docs/getting_started/index.rst

```diff
@@ -111,6 +143,10 @@ Hybrid
 
     pip install -r requirements-dev.txt
 
+5. After installing PARETO, install the open-source solvers provided by the IDAES project::
+
+    idaes get-extensions --verbose
+
 Building Documentation
```

Comment:
Can we add a note about the `make clean` command in the "Building Documentation" section?

Reply:
This has been added to the wiki page: https://github.com/project-pareto/project-pareto/wiki/docs